src/include/stir/unique_ptr.h

```diff
@@ -81,6 +81,10 @@ namespace stir {
 #include <boost/smart_ptr/detail/shared_count.hpp>
 #include <boost/smart_ptr/shared_ptr.hpp>
 
+#ifdef HAVE_NLOHMANN_JSON
+#include <nlohmann/json.hpp>
+#endif
+
```

Comment:
no idea why this is here, but it can't be...

Reply:
Removed by e965da5.